src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -68,6 +68,7 @@ class BankAccountStep extends React.Component {
         this.errorTranslationKeys = {
             routingNumber: 'bankAccount.error.routingNumber',
             accountNumber: 'bankAccount.error.accountNumber',
+            hasAcceptedTerms: 'addDebitCardPage.error.acceptedTerms',
```

Comment:
Let's move the translations to common place from `addDebitCardPage.error.acceptedTerms` `errors.acceptedTerms`. And refactor other places where this error is used.

Reply:
@parasharrajat, do you want me to create a new `errors` object at the main level or shall I put `acceptedTerms` in  `common.error` that is already there?

Reply:
yeah, let's put it in common.error then